src/repository/storage/postgres/orm/sequelize/noteSettings.ts

```diff
@@ -152,6 +152,7 @@ export default class NoteSettingsSequelizeStorage {
       where: {
         noteId: noteId,
       },
+      raw: true,
```

Comment:
What means this line? Why raw?

Reply:
Because by default it returns object with some extra fields such as "_previousDataValues".